### PR TITLE
Update conduct.md

### PR DESCRIPTION
**Pull request by Enorion**
This Pull Request clears the situation about duplicate topics and how they are handled.

### CODE REVIEW

File: conduct.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ possible readability. Long source code posted to the timeline can be difficult
 to read, because of the limited space on Facebook. Topics with large source
 code as their content will be removed from the timeline of the group. Comments
 with large source code as their content will be removed from the content stream.
+Duplicate topics are not encouraged as they fill the group timeline and hinder
+the visibility of other members Topics. Duplicates will be removed from the
```
Comment (petk) [inline conduct.md]: `...of other topics. Duplicates will...`

File: conduct.md [hunk 2]
```diff
@@ -53,6 +53,9 @@ possible readability. Long source code posted to the timeline can be difficult
...
+groups timeline.
```
Comment (petk) [inline conduct.md]: `group timeline.`

**Comment by Enorion** [comment]
Fixed the typos

File: conduct.md [hunk 3]
```diff
@@ -54,8 +54,8 @@ to read, because of the limited space on Facebook. Topics with large source
 code as their content will be removed from the timeline of the group. Comments
 with large source code as their content will be removed from the content stream.
 Duplicate topics are not encouraged as they fill the group timeline and hinder
-the visibility of other members Topics. Duplicates will be removed from the
-groups timeline.
+the visibility of other Topics. Duplicates will be removed from the
```
Comment (petk) [inline conduct.md]: Just one little typo - `topics` in lowercase :)

Comment (Enorion) [reply conduct.md]: I've seen it capitalized in the CoC that's why I used it.. let me quickly unitize it throughout the CoC to lowercase.

Comment (Enorion) [reply conduct.md]: my bad.. I was looking at the start of a sentence :D

**Comment by petk** [comment]
Alright, I'm going to merge this one since the change is good one. In case any of the other two reviewers have comments, let me know.

**Pull request merged**